hcl2template/types.packer_config.go

```diff
@@ -57,6 +57,10 @@ type PackerConfig struct {
 
 	parser *Parser
 	files  []*hcl.File
+
+	force   bool
+	debug   bool
+	onError string
```

Comment:
NIT: do you think it would be a good idea to group these fields together with the only/except ones as the 'fields you can pass from command line' ?

Reply:
@azr are suggestion that these be moved up right under only and except fields with a comment to denote they can be passed on the cli?

Reply:
That's what I understood as well

Reply:
yes !